drivers/SmartThings/zigbee-window-treatment/capabilities/initializedState.yaml

```diff
@@ -0,0 +1,46 @@
+id: stse.initializedstate
+version: 1
+status: proposed
+name: initializedState
+ephemeral: false
+attributes:
+  initializedState:
+    schema:
+      type: object
+      properties:
+        value:
+          type: string
+          enum:
+            - notInitialized
+            - initializing
+            - initialized
+            - initialize

```

Comment:
It is not clear to me what this state means.  A natural reading seems like this would start in `notInitialized` move to `initializing` and end in `initialized`.  `initialize` doesn't seem to fit the naming pattern, and isn't clear where it fits in the flow.